.github/workflows/main.yml

```
@@ -27,7 +27,7 @@ jobs:
           - 2.6
           - 2.7
           - "3.0"
-          - ruby-head
+          - "3.1"
```

I suggest that we keep ruby-head too, so we can spot (and perhaps report on) build errors.
```suggestion
          - "3.1"
          - ruby-head
```